src/classes/Recipe.js

```diff
@@ -10,7 +10,7 @@ class Recipe {
 
   getIngredientNames(ingredientData) {
     return this.ingredients.map(ingredient => {
-      return ingredientData.find(data => ingredient.id === data.id).name.toLowerCase();
+      return ingredientData.find(data => ingredient.id === data.id).name.toLowerCase().split(' ').flat();
```

Comment:
Combined with the .flat() function w/i cookbook this works great! Good thinking, ethan!